vrecord

```diff
@@ -2046,17 +1997,14 @@ _review_all_options(){
         PHASE_VALUE_4="-1*"
     fi
     if [[ "${DEVICE_INPUT_CHOICE}" = "0" ]] ; then
-        _review_option -n -d "ffmpeg" "DECKLINK_UTILITY_CHOICE" "${DECKLINK_UTILITIES[@]}"
         _review_option "VIDEO_INPUT_CHOICE" "${VIDEO_INPUT_OPTIONS[@]}"
         _review_option "AUDIO_INPUT_CHOICE" "${AUDIO_INPUT_OPTIONS[@]}"
         _review_option "VIDEO_BIT_DEPTH_CHOICE" "${VIDEO_BITDEPTH_OPTIONS[@]}"
         _review_option "AUDIO_MAPPING_CHOICE" "${CHANNEL_MAPPING_OPTIONS[@]}"
-        if [[ -z "${ALT_INPUT}" ]] && [[ "${DECKLINK_UTILITY_CHOICE}" == "ffmpeg" ]] ; then
-            _review_option "TIMECODE_CHOICE" "${TIMECODE_OPTIONS[@]}"
-        fi
+        _review_option "TIMECODE_CHOICE" "${TIMECODE_OPTIONS[@]}"
```

Comment:
this seems to break using most files for test inputs - suggest something like

```
if [[ -z "${ALT_INPUT}" ]] ; then
    _review_option "TIMECODE_CHOICE" "${TIMECODE_OPTIONS[@]}"
fi
```

Reply:
done